docs/docs/app-configuration-redwood-toml.md

```diff
@@ -165,7 +165,10 @@ These options allows you to disable the generation of test and story files.
 There are new versions of the framework all the time—a major every couple months, a minor every week or two, and patches when appropriate.
 And if you're on an experimental release line, like canary, there's new versions every day, multiple times.
 
-If you'd like to get notified (at most, once a day) when there's a new version, set `versionUpdates` to include the version tags you're interested in.
+If you'd like to get notified (at most, once a day) when there's a new version, set `versionUpdates` to an array of the version tags you're interested in (tags listed earlier take priority).
+While you can specify any tag, common options include: `latest`, `canary`, `next`, or `rc`.
```

Comment:
```suggestion
While you can specify any tag, common options include: "latest", "canary", "next", or "rc" "experimental", or a valid semver version.
```

These notes exist in the upgrade.js file... if these are all still valid tags, then I will update this PR to the above suggestion. 

```
    .option('tag', {
      alias: 't',
      description:
        '[choices: "latest", "rc", "next", "canary", "experimental", or a specific-version (see example below)] WARNING: "canary", "rc" and "experimental" are unstable releases! And "canary" releases include breaking changes often requiring codemods if upgrading a project.',
      requiresArg: true,
      type: 'string',
      coerce: validateTag,
    })
```
and 

```
  if (!isTagValid) {
    // Stop execution
    throw new Error(
      c.error(
        "Invalid tag supplied. Supported values: 'rc', 'canary', 'latest', 'next', 'experimental', or a valid semver version\n",
      ),
    )
  }
```

Reply:
If you're going with your suggestion Barrett there are a couple of small tweaks I'd do to it. Right now there are two "or", and a missing comma. 

With those corrections it'd look like this:

> While you can specify any tag, common options include: "latest", "canary", "next", "rc", "experimental", or a valid semver version.

I'll let @Josh-Walker-GM comment on the validity of all those choices